aeron-test-support/src/main/java/io/aeron/test/MediaDriverTestUtil.java

```diff
@@ -99,8 +99,8 @@ private void deleteFiles()
         outputFilesByAeronDirectoryName.forEach(
             (aeronDirectoryName, files) ->
             {
-                IoUtil.delete(files.stdout, false);
-                IoUtil.delete(files.stderr, false);
+                IoUtil.delete(files.stdout, true);
+                IoUtil.delete(files.stderr, true);
```

Comment:
Why the change to throw on error? Is it because if one fails the other will fail as well, i.e. due to a running process?

Reply:
The change is the inverse, using `true` will ignore failures on delete.

Reply:
Got it.